src/api/api-categories/api-categories.ts

```diff
@@ -0,0 +1,67 @@
+import axios from "axios";
+import { axiosConfig } from "../../state/state";
+
+export interface ICategory {
+  parentCategoryId: string;
+  name: string;
+  description: string;
+  icon: string;
+  imagePath: string;
+}
+export interface ICategories {
+  name: string;
+  description: string;
+  subcategories: [];
+  uid: string;
+}
+
+export interface ICategoriesQuantity {
```

Comment:
If you need export interface to use somewhere else then consider to move these types to a separate file

Reply:
Please explain to me why I should place such interfaces in a separate file?